tests/e2e-tests/specs/backend/all-reviews.test.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * External dependencies
+ */
+import { switchUserToAdmin } from '@wordpress/e2e-test-utils';
+
+import { visitBlockPage } from '@woocommerce/blocks-test-utils';
+
+const block = {
+	name: 'All Reviews',
+	slug: 'woocommerce/all-reviews',
+	class: '.wc-block-all-reviews',
+};
+
+if ( process.env.WP_VERSION < 5.3 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'skipping all other things', () => {} );
```

Comment:
Not sure what this is doing - is this needed?

Reply:
This disables the tests on old versions of WP where the block is not supported (?).

Reply:
Should be removed for the reviews block.

Reply:
> This disables the tests on old versions of WP where the block is not supported (?).

Aha! That makes sense. Let's change that test name/string on the other block tests so it's clearer :)